montage_mosaic/main.py

```diff
@@ -45,28 +45,33 @@ def main(argv):
         log.info('Target images = {}'.format(" ".join(args.target_images)))
         images = args.target_images
     else:
+        raise LookupError("Must specify the (2D or 3D) images to be mosaicked, each prefixed by '-t '.")
         log.error(
             "Must specify the (2D or 3D) images to be mosaicked, each prefixed by '-t '.")
-        sys.exit()
+            # Still need this for the log file?
 
     # Throw an error if the user provides only one image
     if len(images) < 2:
-        log.error('At least two images must be specified for mosaicking')
-        sys.exit()
+        raise ValueError('At least two images must be specified for mosaicking') 
```

Comment:
I think `raise` and log.error() will both raise an exception. So doing one is enough; I recommend log.error/log.abort

Reply:
I think my latest commits fix things, please re-check x